packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/MessageWithToolResponse.tsx

```diff
@@ -0,0 +1,141 @@
+                      <Flex gap={{ default: 'gapSm' }}>
+                        <FlexItem>
+                          <Label variant="outline" color="blue">
+                            type
+                          </Label>
+                        </FlexItem>
+                        <FlexItem>
+                          <Label variant="outline" color="blue">
+                            properties
+                          </Label>
+                        </FlexItem>
+                        <FlexItem>
+                          <Label variant="outline" color="blue">
+                            label
+                          </Label>
+                        </FlexItem>
+                        <FlexItem>
+                          <Label variant="outline" color="blue">
+                            label
+                          </Label>
+                        </FlexItem>
+                      </Flex>
+                    </FlexItem>
+                  </Flex>
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup
+                style={{ '--pf-v6-c-description-list__group--RowGap': 'var(--pf-t--global--spacer--xs)' } as any}
+              >
+                <DescriptionListTerm>Response</DescriptionListTerm>
+                <DescriptionListDescription>
+                  <ExpandableSection
+                    variant={ExpandableSectionVariant.truncate}
+                    toggleText={isExpanded ? 'show less' : 'show more'}
+                    onToggle={onToggle}
+                    isExpanded={isExpanded}
+                    style={
+                      {
+                        '--pf-v6-c-expandable-section__content--Opacity': '1',
+                        '--pf-v6-c-expandable-section__content--PaddingInlineStart': 0,
+                        '--pf-v6-c-expandable-section__content--TranslateY': 0,
+                        '--pf-v6-c-expandable-section--m-expand-top__content--TranslateY': 0
+                      } as any
+                    }
+                  >
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque nec dignissim turpis, et tristique
+                    purus. Phasellus efficitur ante quis dolor viverra imperdiet. Orci varius natoque penatibus et
+                    magnis dis parturient montes, nascetur ridiculus mus. Pellentesque laoreet, sem ac elementum semper,
+                    lectus mauris vestibulum nulla, eget volutpat massa neque vel turpis. Donec finibus enim eu leo
+                    accumsan consectetur. Praesent massa diam, tincidunt eu dui ac, ullamcorper elementum est. Phasellus
+                    metus felis, venenatis vitae semper nec, porta a metus. Vestibulum justo nisi, imperdiet id eleifend
+                    at, varius nec lorem. Fusce porttitor mollis nibh, ut elementum ante commodo tincidunt. Integer
+                    tincidunt at ipsum non aliquet.
```

Comment:
```suggestion
     Descriptive text about the tool response, including completion status and details on the data that was processed.  
```
Again, not sure if this is the most accurate, but something to replace the lorem